packages/pelagos/src/listInput/ListEntries.js

```diff
@@ -16,14 +19,24 @@ const ListEntries = ({
 	className,
 	highlightKey,
 	list,
+	reorderable,
 	column,
 	getItemKey,
 	getItemName,
 	renderItem,
+	onReorder = moveListItem,
 	onRemoveClick,
 	onHighlightClear,
 }) => {
-	const liveRef = useRef(null);
+	const getElementName = useCallback((element) => getItemName(list[+element.dataset.index]), [list, getItemName]);
+	const updateList = useCallback((fromIndex, toIndex) => onReorder(list, fromIndex, toIndex), [list, onReorder]);
+	const [reorderRef, liveRef] = useReorder(
```

Comment:
Please test with VoiceOver, in my test it didn't read names and positions correctly while reordering.

Reply:
Adding `data-index` here affected the click handler, made the CSS selector there more specific